components/coda/actions/copy-doc/copy-doc.mjs

```diff
@@ -0,0 +1,41 @@
+import coda from "../../coda.app.mjs";
+
+export default {
+  key: "coda_copy-doc",
+  name: "Copy Doc",
+  description: "Creates a copy of the specified Coda doc",
+  version: "0.0.17",
+  type: "action",
+  props: {
+    coda,
+    sourceDoc: {
+      propDefinition: [
+        coda,
+        "sourceDoc",
+      ],
+      description: "The doc ID from which to create a copy.",
+      optional: false,
```

Comment:
I'd recommend removing `optional: true` from the original prop definition. Then you don't have to override it here. There are no clear rules re: optionality but I generally don't like to set the `optional` flag in the original prop definition in the app file, and only override with `optional: true` in specific components. 

The main exception is when the prop is _always_ optional across all components. Then you can set `optional: true` in the prop definition.

Reply:
Issue addressed